File: solutions/php/hamming/3/Hamming.php
<?php
declare(strict_types=1);

function distance(string $strandA, string $strandB): int
{
    return (strlen($strandA) != strlen($strandB)) 
        ? throw new \InvalidArgumentException("DNA strands must be of equal length.") 
        : count(array_diff_assoc(str_split($strandA), str_split($strandB)));
}

File: solutions/python/hamming/1/hamming.py
def distance(strand_a, strand_b):
    if len(strand_a) != len(strand_b): raise ValueError("Strands must be of equal length.")

    counter = 0
    for x in range(len(strand_a)):
        if strand_a[x] != strand_b[x]:
            counter += 1

    return counter
